dashboard/test/controllers/api/v1/assessments_controller_test.rb

```diff
@@ -689,4 +689,23 @@ class Api::V1::AssessmentsControllerTest < ActionController::TestCase
     assert_equal expected_response[level1.id.to_s]['levelgroup_results'],
       actual_response[level1.id.to_s]['levelgroup_results']
   end
+
+  test "section_feedback assert query count" do
+    sign_in @teacher
+    script = create :script
+    lesson_group = create :lesson_group, script: script
+    lesson = create :lesson, script: script, lesson_group: lesson_group
+    weblab_level = create :weblab
+    create :script_level, script: script, levels: [weblab_level], lesson: lesson
+
+    [@student_1, @student_2, @student_3, @student_4].each do |student|
+      create :teacher_feedback, script: script, level: weblab_level, student: student, teacher: @teacher
+    end
+
+    assert_queries 13 do
```

Comment:
Mostly curious, why did you add this test?  (I haven't found these assert_queries tests to be very useful?)

Reply:
This seems to be our current method of keeping track of how many queries we're producing, which is a way of catching n+1s. I noticed that script_level.oldest_active_level was being called for every student in the section and I was wondering if the result of the query was being cached or if we were re-querying to get the level every time. Turns out we were not re-querying, so that's good.
Summary - I was trying to see if there were unnecessary queries being produced, there weren't. I figured I'd leave it so we can track if changes in the future produce more queries than we intend